ReactSkia/Fabric/RSKConversion.h

```diff
@@ -27,16 +15,9 @@ inline SkMatrix RSkTransformTO2DMatrix(const facebook::react::Transform &transfo
           (float)transformMatrix.matrix[15])};
 }
 
-inline SkColor RSkColorConversion(facebook::react::SharedColor sharedColor){ 
```

Comment:
Suggest to change the API name to RSkColorFromSharedColor. Whicj will be in sync with other API "RSkSizeFromSize" and adds more meaning on conversion